lib/livebook/migration.ex

```diff
@@ -1,23 +1,14 @@
 defmodule Livebook.Migration do
   use GenServer, restart: :temporary
 
-  # We version our storage so we can remove migrations in the future
-  # by deleting the whole storage when finding too old versions.
-  #
-  # We also document tables and IDs used in previous versions,
-  # as those must be avoided in the future.
```

Comment:
@josevalim I don't think we need to track these anymore. If we stop using table we should just remove it in a migration, and in the future we can safely use it again.